private/extensions/maven.bzl

```diff
@@ -37,7 +37,7 @@ _install = tag_class(
 
         # What do we fetch?
         "fetch_javadoc": attr.bool(default = False),
-        "fetch_sources": attr.bool(default = True),
+        "fetch_sources": attr.bool(default = False),
```

Comment:
Was this enabled by default by mistake?

Reply:
The simple answer is that this sets the behaviour in the `bzlmod` world to be the same as what we have in the non-bzlmod world.

The slightly longer answer is that we use a logical OR to figure out whether or not to fetch sources. If we set the default to `True`, by default we include all sources, and it would only take one transitive dep in (eg). the default `maven` namespace not to be aware of this to make it something users can't turn off.

The really hand-wavy answer is that I think leaving it set to `True` broke some tests and this was the simplest fix.

The correct answer is to say that if a namespace is specified more than once, and one of those times is the root module, we should do what the root module asks, and only if that's not the case do a logical OR (I think).